src/util.rs

```diff
@@ -91,9 +89,14 @@ where
 {
     Command::new(program)
         .args(args)
+        .stdin(Stdio::null())
+        .stdout(Stdio::null())
+        .stderr(Stdio::null())
         .before_exec(|| unsafe {
-            #[allow(deprecated)]
-            libc::daemon(1, 0);
+            if ::libc::fork() != 0 {
+                ::libc::_exit(0);
+            }
```

Comment:
I can barely read rust and perhaps I'm missing something, but you're not checking that `fork()` succeeds? Meaning, if it fails (granted, very unlikely), you'll terminate the main Alacritty process.

Reply:
This executes in the context of the new process, so it should never lead to Alacritty quitting.

As far as I can tell, if `fork()` doesn't succeed it would just prevent the subprocess from spawning.

I'm not sure *exactly* how `std::process::Command` works, but I'm assuming it forks already. So what we do here is just a second fork to get the double-fork child process zombie fix behavior.

If you don't trust me, it should be possible to test this by just putting `::libc::_exit(0);` in the `before_exec` closure. That should just lead to no subprocess spawning, not to Alacritty quitting.